src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.main_header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
+
+.device_nav {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
```

Comment:
Be consistent with your vertical margins. It seems you're using a right margin for `.nav__item:not(:last-child)`. If there's no specific requirement for vertical margins, choose either top or bottom margin for vertical spacing, but not both.